docs/resources/Channel.md

```diff
@@ -335,6 +335,13 @@ Represents a message sent in a channel within Discord.
 | CONTEXT_MENU_COMMAND                         | 23    | true      |
 | AUTO_MODERATION_ACTION                       | 24    | true*     |
 | ROLE_SUBSCRIPTION_PURCHASE                   | 25    | true      |
+| INTERACTION_PREMIUM_UPSELL                   | 26    | true      |
+| STAGE_START                                  | 27    | true      |
+| STAGE_END                                    | 28    | true      |
+| STAGE_SPEAKER                                | 29    | true      |
+| STAGE_RAISE_HAND                             | 30    | true      |
+| STAGE_TOPIC                                  | 31    | true      |
+| GUILD_APPLICATION_PREMIUM_SUBSCRIPTION       | 32    | false     |
```

Comment:
```suggestion
| STAGE_START                                  | 27    | true      |
| STAGE_END                                    | 28    | true      |
| STAGE_SPEAKER                                | 29    | true      |
| STAGE_RAISE_HAND                             | 30    | true      |
| STAGE_TOPIC                                  | 31    | true      |
```
I think adding the stage message types are fine, but I don't think `INTERACTION_PREMIUM_UPSELL` and `GUILD_APPLICATION_PREMIUM_SUBSCRIPTION` are relevant to apps.

Reply:
It's not about relevance for these types since they can be received regardless.

Reply:
For `INTERACTION_PREMIUM_UPSELL`, I don't believe it can be received (it's ephemeral and unable to be triggered by a bot). i'll change suggestion to include `GUILD_APPLICATION_PREMIUM_SUBSCRIPTION`, cus yah that is similar to role subscriptions.

Reply:
I take that back, the bot is the author of the premium upsell so will encounter. my suggestion is useless, proceeeeeeed

Reply:
Afaik: the stuff for application subscriptions aren't documented at all, it might've been valid to not document it at this point.